.github/workflows/restore-build-test.yml

```
@@ -11,9 +11,14 @@ on:
         type: string
         required: true
         description: JSON array of project names to test (e.g., '["ProjectA", "ProjectB"]')
+    secrets:
+      GIST_SECRET:
+        required: true
+        description: Secret for accessing Gist API to create badges
 
 permissions:
   contents: read
+  gists: write
```

actionlint (1.7.7)

unknown permission scope "gists". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)